Examples/cChocoFeatureExample.ps1

```diff
@@ -0,0 +1,28 @@
+
+
+configuration ChocoFeatures {
+
+    Import-DscResource -ModuleName cChoco
+
+    Node 'localhost' {
+
+        cChocoFeature allowGlobalConfirmation {
+
+            FeatureName = "allowGlobalConfirmation"
```

Comment:
That's frustrating and verbose. I asked some experts on the subject just to be entirely sure and they indicated the same. It sucks because that kind of thing is entirely possible with Puppet (and I'm sure with Chef as well).